### PR TITLE
fix: use any to unblock typescript compilation bug

### PR DESCRIPTION
**Pull request by stephenplusplus**
`as any` saves the day.

### CODE REVIEW

File: test/publisher/message-queues.ts [hunk 1]
```diff
@@ -295,7 +295,7 @@ describe('Message Queues', () => {
       });
 
       it('should cancel any pending publish calls', () => {
-        const fakeHandle = (1234 as unknown) as NodeJS.Timer;
+        const fakeHandle = (1234 as unknown) as any;
```
Comment (alexander-fenster) [inline test/publisher/message-queues.ts:298]:
```suggestion
        // eslint-disable-next-line @typescript-eslint/no-explicit-any
        const fakeHandle = (1234 as unknown) as any;
```

**Comment by feywind** [comment]
Linking to the issue. https://github.com/googleapis/nodejs-pubsub/issues/1009

**Comment by stephenplusplus** [comment]
@alexander-fenster looks like I made everything confusing by accepting your suggestion :X

**Comment by alexander-fenster** [comment]
@stephenplusplus I fixed it! It's weird, I didn't know I will be shown as a co-author if I just suggest a change. Apparently, it used my non-Google email for the commit, or something like that.

**Comment by stephenplusplus** [comment]
Thank you! That is quite weird. Oh well, thanks for catching that :)

**Pull request merged**